.github/workflows/generator_generic_slsa3.yml

```diff
@@ -234,7 +238,10 @@ jobs:
           # number of subjects based on in-toto attestation bundle file naming conventions.
           # See: https://github.com/in-toto/attestation/blob/main/spec/bundle.md#file-naming-convention
           # NOTE: The attest commmand outputs the provenance-name and provenance-sha256
-          "$GITHUB_WORKSPACE/$BUILDER_BINARY" attest --subjects-filename "${SUBJECTS_FILENAME}" -g "$untrusted_prov_name"
+          output=$("$GITHUB_WORKSPACE/$BUILDER_BINARY" attest --subjects-filename "${SUBJECTS_FILENAME}" -g "$untrusted_prov_name")
+          echo "$output"
+          uuid=$(echo "$output" | grep -oP 'UUID \K[0-9a-f]{80}')
```

Comment:
nonblocker: it seems the rekor UUID is meant to be 64 hex digits
 - https://github.com/sigstore/rekor/blob/5e502df5603712e553ab4a077793bdece66d4dbc/pkg/api/error.go#L42

Reply:
The length of the UUID generated in "create and sign provenance" is 80.

https://search.sigstore.dev/?uuid=24296fb24b8ad77a037b175b6a98db3d318ae8d94428f5be89eb07c8a6b70b58b54bdd32c7b15fb6

https://docs.sigstore.dev/logging/sharding/#identifier-definitions-entryid-uuid-logid-log-index

So, I think the UUID used as a search parameter in sigstore is actually an entry ID.

Reply:
The additional prefix digits are for distinguishing which log "shard" hosts the content.